nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -595,41 +655,172 @@ sub multiLineList {
+  # by default).
+  #
+  # environment.systemPackages = [ pkgs.apulse ];
 
   # Enable the X11 windowing system.
+  #
   # services.xserver.enable = true;
   # services.xserver.layout = "us";
   # services.xserver.xkbOptions = "eurosign:e";
-
+  #
   # Enable touchpad support.
   # services.xserver.libinput.enable = true;
 
-  # Enable the KDE Desktop Environment.
-  # services.xserver.displayManager.sddm.enable = true;
-  # services.xserver.desktopManager.plasma5.enable = true;
+  # Enable graphical acceleration for NVIDIA GPUs (requires unfree
+  # packages, see above).
+  #
+  # services.xserver.videoDrivers = [ "nvidia" ];
+  #
+  # For older NVIDIA GPUs you can use the following.
+  #
+  # services.xserver.videoDrivers = [ "nvidiaLegacy340" ];
+  # or
+  # services.xserver.videoDrivers = [ "nvidiaLegacy304" ];
+  # or
+  # services.xserver.videoDrivers = [ "nvidiaLegacy173" ];
+  #
+  # Acceleration on modern AMD/ATI and Intel GPUs will work with free
+  # drivers installed out of the box, but you might still want to
+  # override the default value of this option just to save some space
+  # by not installing the drivers you don't need. E.g.
+  #
+  # services.xserver.videoDrivers = [ "intel" ];
+  # or
+  # services.xserver.videoDrivers = [ "ati" ];
+  #
+  # Some older ATI GPUs might need the following to get the
+  # acceleration working (requires unfree packages, see above).
+  #
+  # services.xserver.videoDrivers = [ "ati_unfree" ];
+  # or
+  # services.xserver.videoDrivers = [ "amdgpu" ];
+  # or
+  # services.xserver.videoDrivers = [ "amdgpu-pro" ];
+
+  # Enable support for 32-bit OpenGL libraries on x86_64 which is
+  # useful if you want to run 32-bit Windows games under Wine.
+  #
+  # hardware.opengl.driSupport32Bit = true;
+  # environment.systemPackages = with pkgs; [ wine winetricks ];
+
+  # Some of the drivers above might not work with the current Linux
+  # kernel, so you might end up overriding that too.
+  #
+  # boot.kernelPackages = pkgs.linuxPackages_4_9;
 
   # Define a user account. Don't forget to set a password with ‘passwd’.
-  # users.extraUsers.guest = {
+  #
+  # users.users.me = {
   #   isNormalUser = true;
   #   uid = 1000;
+  #   extraGroups = [ "input" "audio" "video" "cdrom" ]
+  #              ++ [ "wheel" ]; # for sudo
+  #   openssh.authorizedKeys.keyFiles = [ /etc/nixos/id_rsa.pub ];
   # };
 
-  # This value determines the NixOS release with which your system is to be
-  # compatible, in order to avoid breaking some software such as database
-  # servers. You should change this only after NixOS release notes say you
-  # should.
-  system.stateVersion = "${\(qw(@release@))}"; # Did you read the comment?
+  # Enable the KDE Desktop Environment.
+  #
+  # services.xserver.displayManager.sddm.enable = true;
+  # services.xserver.desktopManager.plasma5.enable = true;
+
+  # Or maybe you want to enable Xfce Desktop Environment and Slim
```

Comment:
:-1: on the addition of xfce / slim and the following, they'll need to find the full list of options somehow, and this file isn't in the business of describing every use case. We should be teaching them to fish here.